### DISCUSSION TITLE
Exporting? Tutorial?

### DISCUSSION THREAD

**Discussion by loganbush789** [DependencyTrack/dependency-track#948] [Q&A]
Is there a way to export vulnerability findings? Also, is there a tutorial posted anywhere?

**Comment by stevespringett** [comment]
The [Findings API](https://docs.dependencytrack.org/integrations/file-formats/) may be what you're looking for. The API has the following endpoints:
`/api/v1/finding/project/{uuid}`
`/api/v1/finding/project/{uuid}/export`

**Reply by kpopple** [reply]
Example - from a terminal issue: curl -x "GET" "http://<your server:port>/api/v1/finding/project/<project uuid> -H "X-API-Key: <api-key>"
